deployments/liqo/templates/webhooks/job-patch/job-create-secret.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
   labels:
     {{- include "liqo.labels" $cfg | nindent 4 }}
 spec:
-  ttlSecondsAfterFinished: 0
+  ttlSecondsAfterFinished: 600
```

Comment:
Maybe would be better a lower value, like 150 seconds. Same for the next one.

Reply:
@aleoli What's your opinion here?

Reply:
150 will be fine since it seems to be the value generally used